src/main/java/dk/cngroup/lentils/entity/HintTakenKey.java

```diff
@@ -0,0 +1,54 @@
+package dk.cngroup.lentils.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import java.io.Serializable;
+import java.util.Objects;
+
+@Embeddable
+public class HintTakenKey implements Serializable {
```

Comment:
*thumbsup* To jsem ani nevěděl, že ten klíč jde napsat takhle jednoduše, paráda.